core/src/sidebar/sidebar.module.css

```diff
@@ -0,0 +1,33 @@
+.username {
+	font-weight: 600;
+	font-size: 18px;
+	line-height: 24px;
+	color: var(--gray-8);
+	align-self: stretch;
+}
+
+.role {
+	font-size: 12px;
+	line-height: 18px;
+	color: var(--gray-3);
+	text-transform: capitalize;
+	align-self: stretch;
+}
+
+.header {
+	padding: 24px 0;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+}
+
+.header:hover,
+.header:active {
+	cursor: pointer;
```

Comment:
Should be removed since the header does not involve any action.
Or you could do `user-select: none;`